resources/Categorie_fallacie.logiche.md

```diff
@@ -0,0 +1,185 @@
+APPELLARSI ALLA “RAGIONE”
+
+*   chiamare in gioco un' autorita' anonima
+
+usare prove da esperti o studi sconosciuti, i cosiddetti “dicono che...”: “Dicono che OS X sia un sistema inattaccabile dai virus”;
+
```

Comment:
Non capisco bene la semantica. C'e' un elenco puntato e poi sotto delle frasi scollegate? Forse non sarebbe meglio che le frasi fossero sulla stessa linea, ad esempio:

* chiamare in gioco un' autorita' anonima: usare prove da esperti o studi sconosciuti, i cosiddetti “dicono che...”: “Dicono che OS X sia un sistema inattaccabile dai virus”;

?